src/managers/GuildMemberManager.js

```diff
@@ -136,6 +137,45 @@ class GuildMemberManager extends BaseManager {
     return this._fetchMany(options);
   }
 
+  /**
+   * Edits a member of the guild.
+   * <info>The user must be a member of the guild</info>
+   * @param {UserResolvable} member The member to edit
+   * @param {GuildMemberEditData} data The data to edit the member with
+   * @param {string} [reason] Reason for editing this user
+   * @returns {Promise<GuildMember>}
+   */
+  async edit(member, data, reason) {
+    const id = this.client.users.resolveID(member);
+    if (!id) throw new TypeError('INVALID_TYPE', 'user', 'UserResolvable');
+
+    // Clone the data object for immutability
+    const _data = { ...data };
+    if (_data.channel) {
+      _data.channel = this.guild.channels.resolve(_data.channel);
+      if (!_data.channel || _data.channel.type !== 'voice') {
+        throw new Error('GUILD_VOICE_CHANNEL_RESOLVE');
+      }
+      _data.channel_id = _data.channel.id;
+      _data.channel = undefined;
+    } else if (data.channel === null) {
+      _data.channel_id = null;
+      _data.channel = undefined;
+    }
+    if (_data.roles) _data.roles = _data.roles.map(role => (role instanceof Role ? role.id : role));
+    let endpoint = this.client.api.guilds(this.guild.id);
+    if (id === this.client.user.id) {
+      const keys = Object.keys(_data);
+      if (keys.length === 1 && keys[0] === 'nick') endpoint = endpoint.members('@me').nick;
+      else endpoint = endpoint.members(id);
+    } else {
+      endpoint = endpoint.members(id);
+    }
+    const d = await endpoint.patch({ data: _data, reason });
+
+    return this.cache.get(id)?._clone().patch(d) ?? this.add(d, false);
```

Comment:
`GuildMember#_patch` does not return anything.

```suggestion
    const clone = this.cache.get(id)?._clone();
    clone?.patch(d);
    return clone ?? this.add(d, false);
```

Alternatively you could make `_patch` return `this`.
(Only 2 of 31 of your classes seem to do that though, not that I see an issue with that)

Reply:
Fixed in #bbfd002